tests/vmi_configuration_test.go

```diff
@@ -2137,17 +2137,21 @@ var _ = Describe("[sig-compute]Configurations", decorators.SigCompute, func() {
 		var dataVolume *cdiv1.DataVolume
 
 		BeforeEach(func() {
-			var err error
 			if !checks.HasFeature(virtconfig.HostDiskGate) {
 				Skip("Cluster has the HostDisk featuregate disabled, skipping  the tests")
 			}
 
-			dataVolume, err = createBlockDataVolume(virtClient)
-			Expect(err).ToNot(HaveOccurred())
-			if dataVolume == nil {
+			sc, foundSC := libstorage.GetBlockStorageClass(k8sv1.ReadWriteOnce)
+			if !foundSC {
 				Skip("Skip test when Block storage is not present")
 			}
 
+			dataVolume := libdv.NewDataVolume(
```

Comment:
I believe you want to assign to the existing `dataVolume` var and not initialize a new one

Reply:
Fixed, thanks neglected this PR for the past 2 weeks